nims-notes-things/src/Form.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import TextArea from "./TextArea";
+import Input from "./Input";
+
+const Form = (props) => {
+  const [formData, setFormData] = useState({
+    name: "",
+    date: "",
+    note: "",
+  });
+
+  const [empty, setFormEmpty] = useState({
+    name: false,
+    date: false,
+    note: false,
+  });
+
+  const handleOnChange = (e, key) => {
+    const newForm = {
+      ...formData,
+      [key]: e.target.value,
+    };
+    setFormData(newForm);
+  };
+
+  const handleOnBlur = (value, key) => {
+    setFormEmpty((previousState) => {
+      return {
+        ...previousState,
+        [key]: value === "",
+      };
+    });
+    //console.log(empty);
+  };
+
+  return (
+    <form onSubmit>
```

Comment:
You should probably have this as a styled component `styled.form` with all your stylings

Reply:
Also onSubmit needs to point to a function that is triggered when you submit the form

Reply:
My styling is on another branch